pkgs/applications/science/misc/sasview/default.nix

```diff
@@ -0,0 +1,45 @@
+{lib, fetchgit, gcc, python}:
+
+python.pkgs.buildPythonApplication rec {
+  name = "sasview-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed. Instead use `pname` for just the name.